cmd/contextconfig.go

```diff
@@ -32,16 +32,42 @@ type ContextConfig struct {
 }
 
 func NewContextConfig(storeRoot string) (*ContextConfig, error) {
```

Comment:
Step 2 is to make this a map with different roots for each operator, but reviewing step 1 only, wanted a comment to remind myself

Reply:
yes, we need to make this a map that is stored as a pref, then we can show the ones we know about.